### ISSUE TITLE
cbwr_get() fails after cbwr_set() succeeds

### ISSUE THREAD

**Issue by aldanor** [IntelPython/mkl-service#7]
Is this expected?

```python
>>> mkl.cbwr_set('avx2')
'success'

>>> mkl.cbwr_get()
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
ValueError: Variable can not be None
Exception ignored in: 'mkl._py_mkl_service.__mkl_str_to_int'
ValueError: Variable can not be None
'err_invalid_input'
```

**Comment by aldanor** [comment]
Apologies, my bad - apparently you have to do `cbwr_get("all")` which is a bit unobvious and non-Pythonic I guess - because the default argument (`None`) is invalid.

Closing this, anyway.

**Comment by oleksandr-pavlyk** [comment]
I fixed this in pull request #9 as well. `mkl.cbwr_get()` is now equivalent to `mkl.cbwr_get("all")`.
